lib/web/README.md

```diff
@@ -16,13 +16,13 @@ follow the steps in [binaryen](https://github.com/WebAssembly/binaryen#building)
 
 To compile all features for WASM target:
 ```bash
-wasm-pack build --target web
+wasm-pack build --target web --out-name didkit-wasm
 ```
 
 To compile all features for ASM target:
 ```bash
-wasm-pack build --target bundler
-BINARYEN_ROOT/bin/wasm2js --pedantic -o didkit_wasm_bg1.js didkit_wasm_bg.wasm
+wasm-pack build --target bundler --out-name didkit-asm
+BINARYEN_ROOT/bin/wasm2js --pedantic -o didkit-asmjs.js didkit-asmjs.wasm
 ./repack.sh
 npm install
 npm run build
```

Comment:
These should be run from `lib/web/loader/asmjs`, so maybe add a `cd loader/asmjs` since we already assume the working directory to be `lib/web`.